src/styles/home_nav.css

```diff
@@ -0,0 +1,225 @@
+.home-nav {
+  z-index: 1;
+  overflow: hidden;
+}
+.home-nav a {
+  text-decoration: none;
+  letter-spacing: 0.15em;
+  text-transform: uppercase;
+}
+
+.home-nav-menu a {
+  transition: color 0.3s ease;
+  color: #9da0a7;
+  font-size: .8em;
+  font-weight: 400;
+}
+
+.home-nav a:hover {
+  cursor: pointer;
+  color: #fff;
+}
+
+.nav-container {
+  padding-left: 40px;
+  padding-right: 40px;
+  background-color: #3d4351;
+  font-family: 'Catamaran', sans-serif;
+  font-weight: 300;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  overflow: hidden;
+}
+
+.logo{
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: 14px 0 14px 0;
+  width: 210px;
+  height: 36px;
+}
+
+.logo-green-square {
+  background-color: #8dc63f;
+  height: 33px;
+  width: 33px;
+  min-width: 33px;
+  display: flex;
+  align-items: center;
+}
+
+.logo-green-square div {
+  font-size: 2em;
+  color: #3d4351;
+  font-weight: 200;
+  position:absolute;
+  left:46px;
+  top:9px;
+}
+
+.logo img {
+  max-height: 35px;
+}
+
+@media only screen and (max-width: 1200px) {
+  .logo-type {
+    visibility: hidden;
+  }
+}
+
+@media only screen and (max-width: 1030px) {
+  .logo-type {
+    visibility: visible;
+  }
+}
+
+.logo-type {
+  margin-left: 10px;
+  max-height: 15px !important;
+}
+
+.home-nav-right{
+  flex-grow: 1;
+}
+
+.home-nav-menu {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: flex-end;
+  flex-direction: row;
+  align-items: center;
+  max-height: 64px;
+  overflow: hidden;
+}
+
+.home-nav-menu a {
+  margin-left: 25px;
+}
+
+.nav-dropdown span {
+  display: block;
+}
+
+@media only screen and (max-width: 720px) {
+  .home-nav-right {
+    visibility: hidden;
+  }
+}
+
+@media only screen and (max-width: 1120px) {
+  .home-nav-right {
+    visibility: hidden;
+  }
+}
+
+.nav-dropdown {
+  display: inline-block;
+}
+
+.nav-dropdown div {
+  margin-right:10px
+}
+
+.more-triangle {
+  border-bottom:5px solid transparent;
+  border-left: 5px solid transparent;
+  border-right: 5px solid transparent;
+  border-top: 5px solid ;
+  max-width: 0px;
+  max-height: 0px;
+  position: absolute;
+  right: 520px;
+  top: 29px;
+  z-index: 15;
+}
+
+.home-nav-triangle {
+  border-bottom:10px solid white;
+  border-left: 10px solid transparent;
+  border-right: 10px solid transparent;
+  border-top: 10px solid transparent;
+  max-width: 0px;
+  max-height: 0px;
+  position: absolute;
+  right: 545px;
+  top: 38px;
+  z-index: 15;
+}
+
+.home-nav-dropdown {
+  position: absolute;
+  background-color: #fff;
+  min-width:100px;
+  z-index: 10;
+  border-radius: 3px;
+  right: 458px;
+  top: 58px;
+  box-shadow: 0 0 10px rgba(0,0,0,.1);
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```